dist/config.yaml

```diff
@@ -0,0 +1,116 @@
+project_name: scorecard
```

Comment:
Where is this file being used?

Reply:
must;ve accidentally committed while on autopilot.